### ISSUE TITLE
Can't connect my new android phone to wallabag.it

### ISSUE THREAD

**Issue by jeanmarie18** [wallabag/android-app#729]
# Issue details

### Duplicate?
no

### Actual behaviour
I can't connect to my usual wallabag (paid) instance, at https://app.wallabag.it
I checked several times the user and password.
I created the new client and input it in the app with its secret and checked it it correctly typed.

### Expected behaviour
I should be able to login and sync my articles on my new device

# Steps to reproduce the issue
1. launch wallabag app (android 8.1)
2.
3. 

# Environment details
- **wallabag app version**: 2.0.2
- **wallabag app installation source** (e.g. Gplay, F-Droid, manual): Play Store
- **Android OS version**: 8.1.0
- **Android ROM** (e.g. stock, LineageOS, SlimRom,…): OxygenOS 5.1.3
- **Android hardware**: Oneplus 6
- **wallabag server version**: 2.3.2

# Logs
## wallabag server
I have no logs as I paid the service at https://app.wallabag.it

## Web server
same as above

# Your experience with wallabag Android app
I currently have my articles on another Android device (7.1.2) and have no problem. I just want to be able to read them on my new device (client).

**Comment by Strubbl** [comment]
Thank you for your report. 

Do you get any error message? Which one? When exactly does the error occur?

**Comment by jeanmarie18** [comment]
I am sorry.
I just tried again this morning and directly went to parameters and choosed "Lancer l'assistant de connexion": it did all the job!! and now it works! (I just had to choose wallabag.it, type my username and password, and that was it)
At first I did not launch this option, which is the first of the "Parameters" menu... I directly went to "Connexion" (which is the second one) and entered manually all the fields ... however I don't understand why this did not work.
Thank you anyway for your fast answer.
